control.py

```diff
@@ -17,69 +22,215 @@ def hello():
         return request.json
 
 
-@app.route("/diretores", methods=["GET", "POST", "DELETE"])
-def diretor():
+@app.route("/diretores", methods=["GET", "POST"])
+def diretor_GetPost():
```

Comment:
Pra funções no python, a não ser que exista algum padrão externo necessário, utilize snake case
```suggestion
def diretor_get_post():
```